plejd/mqtt.js

```diff
@@ -164,6 +176,12 @@ class MqttClient extends EventEmitter {
         getConfigPath(device),
         JSON.stringify(payload)
       );
+      setTimeout(() => {
+        self.client.publish(
+          getAvailabilityTopic(device),
+          "online"
+        );
+      }, 2000);
```

Comment:
After initial discovery there's may be a slight delay before Home Assistant is ready to accept any data about it, hence the delay before the availability is reported.